qtawesome/iconic_font.py

```diff
@@ -725,7 +725,17 @@ def _install_fonts(self, fonts_directory, system_wide=False):
                             # Possible permission error when trying to remove
                             # a font which potentially is already in use
                             # See spyder-ide/qtawesome#236
-                            continue
+                            pass
+                        # Regardless of the font file being actually removed
+                        # since the load failed we need to return here where
+                        # the font file is originally located. With that a
+                        # `FontError` can be eventually raised with useful
```

Comment:
Instead of returning `fonts_directory` below, why don't we raise a `FontError` there, saying that it was not possible to load a specific font? The error message could also use `get_fonts_info` to tell users what fonts they need to install manually and where they are located.

Reply:
After giving it some thought, if we raise `FontError` here the fallback logic to use system wide fonts will never be reached (since then always an error here will be raised). Maybe we should show instead a warning here and only in case we are unable to detect the font that failed being loaded is available from the system fonts directory? What do you think @ccordoba12 ?

Reply:
Last commit implements the idea above but let me know what you think

Reply:
Ok, that looks better to me. But then I think we don't need to return `fonts_directory` below, but a simple `return`. And the comment you added here should be reworded, right?